recipes/libtiff/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set version = "4.0.6" %}
+
+package:
+    name: libtiff
+    version: {{ version }}
+
+source:
+    fn: tiff-{{ version }}.tar.gz
+    url: ftp://ftp.remotesensing.org/pub/libtiff/tiff-{{ version }}.tar.gz
+    md5: d1d2e940dea0b5ad435f21f03d96dd72
+    patches:
+        - def_snprintf.patch  # [win]
```

Comment:
From the original recipe in conda-recipes, but I see that is a similar fix to what we did to gdal.